src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -324,4 +324,10 @@
   <data name="SerializerOptionsImmutable" xml:space="preserve">
     <value>Serializer options cannot be changed once serialization or deserialization has occurred.</value>
   </data>
+  <data name="SerializerPropertyNameConflict" xml:space="preserve">
+    <value>The property '{0}.{1}' has the same name as a previous property based on naming or casing policies.</value>
```

Comment:
Do we need to be this specific about the policies list?

How about something like `The property '{0}.{1}' collides with an existing property name.`

Reply:
From a type system the property can't collide, so the message is trying to make it more clear.